http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -188,116 +192,73 @@ USER_SCRIPT_OUTPUT_URI=$(userScriptOutputUri)
 JUPYTER_NOTEBOOK_FRONTEND_CONFIG_URI=$(jupyterNotebookFrontendConfigUri)
 CUSTOM_ENV_VARS_CONFIG_URI=$(customEnvVarsConfigUri)
 RSTUDIO_LICENSE_FILE=$(rstudioLicenseFile)
+GPU_ENABLED=$(gpuEnabled)
 
-mkdir -p /work
-mkdir -p /certs
+CERT_DIRECTORY='/var/certs'
+DOCKER_COMPOSE_FILES_DIRECTORY='/var/docker-compose-files'
+WORK_DIRECTORY='/mnt/disks/work'
+GSUTIL_CMD='docker run --rm -v /var:/var gcr.io/google-containers/toolbox:20200603-00 gsutil'
+GCLOUD_CMD='docker run --rm -v /var:/var gcr.io/google-containers/toolbox:20200603-00 gcloud'
+DOCKER_COMPOSE='docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v /var:/var docker/compose:1.29.1'
+
+mkdir -p ${WORK_DIRECTORY}
+mkdir -p ${CERT_DIRECTORY}
+mkdir -p ${DOCKER_COMPOSE_FILES_DIRECTORY}
 
 log 'Formatting and mounting persistent disk...'
 
 # Format and mount persistent disk
-export DISK_DEVICE_ID=$(lsblk -o name,serial | grep 'user-disk' | awk '{print $1}')
-# Only format disk is it hasn't already been formatted
+# Fix this to `sdb`. We've never seen a device name that's not `sdb`,
```

Comment:
Fair enough :)

Reply:
more important reason is becuz the original cmd doesn't work anymore :) we can improve this if we realize there's a need